src/components/sections/section.tsx

```diff
@@ -33,6 +33,7 @@ export default function SectionGroup({
     <View style={containerStyle}>
       {React.Children.map(children, (child, index) => {
         if (!React.isValidElement(child)) return child;
+        if (child == null) return child;
```

Comment:
Trying to catch `undefined` as well?

Reply:
Yeah, can do that. It was a specific use case where returning `null` from component is a common use case. But as you say, undefined can also be a thing.

Reply:
Was a reference to it being a falsy check `==`. Was wondering myself if `undefined` is a case.

Reply:
Yes. That's right. Everything is as it should